src/components/AssetItem/AssetItem.tsx

```diff
@@ -10,16 +10,16 @@ import {Boundary, Placeholder, Text, TokenIcon} from '..'
 
 export type AssetItemProps = {
   tokenInfo: TokenInfo
```

Comment:
instead of passing around tokenInfo, pass around tokenId or amount
and `useTokenInfo({ wallet, tokenId })` when you need to use it